index.js

```diff
@@ -9,8 +10,57 @@ async function getParserRules({ octokit, owner, repo, path }) {
   return JSON.parse(content);
 }
 
+async function writeStatusToIssue({ octokit, owner, repo, issue, status }) {
+  return await octokit.issues.createComment({
+    owner: owner,
+    repo: repo,
+    issue_number: issue.number,
+    body: status
+  });
+}
+
+function __generateMessageBody(messageSuffix, actions) {
```

Comment:
In JS this is not a pattern that is followed. All functions are module scoped and inaccessible from elsewhere. The only way to make them available to other modules is to export them using `modules.exports`